wsat-simple/configure-server.cli

```diff
@@ -0,0 +1,17 @@
+
+# Batch script to configure XTS subsystem
+
+
+/extension=org.jboss.as.xts:add
+
+batch
+
+# Creating the xts subsystem
+/subsystem=xts/:add(host=default-host, url="http://${jboss.bind.address:127.0.0.1}:8080/ws-c11/ActivationService", default-context-propagation=true )
+
+# Run the batch commands
+run-batch
+
+# Reload the server configuration
+#reload
```

Comment:
That is a good question! I took the example from the script Eduardo linked to me: https://github.com/wildfly/quickstart/blob/main/servlet-security/configure-server.cli . Since I am not an expert about it I did not change that line. I would agree to uncomment it if you want so.

Reply:
Generally you do reload if needed :-)